### ISSUE TITLE
Bibliography modern filter: sorteringsordningen är lite skum

### ISSUE THREAD

**Issue by Maanin00** [humlab-sead/sead_browser_client#243]
Först kommer ett antal tomma poster, sen kommer några poster i oordning innan siffror och sen A inleder den rätta sorteringen. Det är även ett antal poster i slutet av listan som sorteras annorlunda exempelvis http-poster hamnar inte under h och diakritiska tecken läggs sist. En större fråga väcks här: hur ska diakritiska tecken sorteras? När är såna en egen bokstav och när är det en variant på en bokstav? Se https://sv.wikipedia.org/wiki/Diakritiskt_tecken

![bild](https://user-images.githubusercontent.com/100921064/228159664-0dff2a82-64be-430a-864b-49b7958cbeec.png)

**Comment by johanvonboer** [comment]
Har tittat lite på detta och det beror på 'skräpigt' data i databasen.

Ips typographus... är en sträng som börjar med en tabb. Resten av strängarna ner till 'British... börjar med ett mellanslag. Därav att sorteringsordningen blir som den blir.

Dessa tabbar och mellanslag syns dock inte eftersom de strippas ut från själva renderingen.

Att http-länkar hamnar långt ner skulle jag tro beror på att de börjar med liten bokstav.

Jag kan fixa dessa två saker på klientsidan (de borde dock fixas även i databasen längre fram).

Dock finns det andra poster som är mer problematiska, t.ex. den här posten (ArtDatabanken):
![image](https://user-images.githubusercontent.com/34707290/230386278-1e1abe7b-b87f-48fe-8f1e-f247d430dff9.png)
Ser egentligen ut så här i databasen, d.v.s. den är inlagd som en tagg:
![image](https://user-images.githubusercontent.com/34707290/230386089-f4747874-7dee-4fef-8eed-94653c91d611.png)

Och hamnar därför konstigt i sorteringsordningen. Detta är saker som antagligen bör fixas i databasen.

**Comment by Maanin00** [comment]
Ska jag göra ett nytt ärende för ändring i db? Det gäller tbl_biblio, va?

**Comment by johanvonboer** [comment]
Ja, gärna. Man skulle nog behöva göra något enkelt script för att gå igenom den tabellen och ta bort alla eventuella \n och \t framför strängarna och ta bort tagg-tecknen "<" ">" där de finns.

**Comment by johanvonboer** [comment]
It was decided that @visead will make a new view for fetching the correct data.

**Comment by johanvonboer** [comment]
We also need a tooltip showing the complete title
